docs/gettingStarted/setupAlchemyDevMode.md

```diff
@@ -1 +1,168 @@
+  
+
+  Choose from one of the following setup for testnet to start playing/integrating features to Alchemy:
+  
+  - [Use daostack rinkeby subgraph](#use-daostack-rinkeby-subgraph)
+  - [Run graph-node locally](#run-graph-node-locally)
+
+  
+  NOTE: 
+  
+  1. Alchemy only shows `daos` that are registered via `DAOregistry` and tracked by daostack subgraph for the respective network. You can send the `.json` of your DAO details to us (contact Nave Rachman, telegram: @NaveRachman) and we will help you.
+
+  2. Since above process of registering DAO takes up to 24hrs in following section we provide way to hack it during development
+
+### Use daostack rinkeby subgraph
+
+  Choose this if,
+
+  - using rinkeby testnet
+  - working with existing whitelisted DAOs on daostack subgraph
+
+  Make following changes:
+
+  1. Update `docker-compose.yml`
+
+     - remove link to `graph-node` in service `alchemy`
+     - remove services `graph-node`, `ipfs`, `postgres4graphnode` and `ganache`
+
+  2. Update `webpack.docker.config.js`
+
+     - Change `NODE_ENV` from `development` to `staging`
+
+  3. Build and run
+        
+          docker-compose build
+          docker-compose up -d
+
+### Run graph-node locally
+
+  Choose this if,
+
+  - using any of the already supported schemes by client & alchemy
+  - playing with DAO you just deployed to any of the testnet and not yet whitelisted by daostack and registered with DAOregistry
+
+  NOTE: 
+   
+   1. You can also deploy subgraph to graph explorer. `Step 1 & 2` will be replaced by [deploy to graph-explorer](https://github.com/daostack/subgraph#deploy-subgraph)
+   and `Step 4` will be updated to corresponding url
+
+   2. Alchemy only shows `daos` that are registered via `DAOregistry`. In order to bypass this during development, make changes in `src/components/Daos/DaosPage.tsx`:
+      
+        // original
+
+        arc.daos({ where: { name_not_contains: "Genesis Alpha", register: "registered" }, orderBy: "name", orderDirection: "asc"})
+
+        // changed to
+
+        arc.daos({ where: { name_not_contains: "Genesis Alpha" }, orderBy: "name", orderDirection: "asc"})
+
+  Make following changes:
+
+  1. Clone subgraph repo and start-graph node locally
+
+        git clone git@github.com:daostack/subgraph.git
+        cd subgraph
+        npm i
+
+  2. Setup `.env` file and run rinkeby graph node
+
+        // Following are example values please change for customization
+
+        network="rinkeby"
+        subgraph="daostack"
+        postgres_password="letmein"
+        ethereum_node="https://rinkeby.infura.io/v3/e0cdf3bfda9b468fa908aa6ab03d5ba2"
+
+        npm run docker:run-rinkeby 
+
+  2. Update your DAO details in `daos/rinkeby/<DAO-Name>.json` and deploy subgraph
+
+        npm run deploy '{  "migrationFile" : "../migration.json" }'
+        
+  3. Go back to directory `alchemy`
+        
+  4. Update `docker-compose.yml`
+
+        - remove link to `graph-node` in service `alchemy`
```

Comment:
OR, again, just ignore the whole docker setup altogether, and do something like:

- run the graphql server as in step 1
- run alchemy with`npm run start`
- if you want to save profile info and such, run the alchemy-server using `docker-compose run alchemy-server` (but most functions also work without)